frontend/packages/console-shared/src/components/getting-started/GettingStartedCard.tsx

```diff
@@ -51,30 +53,43 @@ export const GettingStartedCard: React.FC<GettingStartedCardProps> = ({
       direction={{ default: 'column' }}
       grow={{ default: 'grow' }}
       className="ocs-getting-started-card"
+      data-test={`column ${id}`}
```

Comment:
I wasn't sure how to call this three columns.

The component is called card (which was maybe a mistake), but I added a "card" data-test attribute to complete "getting started" card. This matches more the visual part of the component and not the component name.

Reply:
Perhaps this class should have been named `Item` or `Entry` instead of `Card`.

Reply:
Can you please change it to `data-test-id={`column ${id}`}`

Reply:
Yep, renaming the `GettingStartedGrid` to `GettingStartedCard` and `GettingStartedCard` to `GettingStartedItem` make sense. But if we would rename this to `GettingStartedItem` this could confuse because the links are also called items internally and use `data-test="item"`

So fetching all items would return the columns and the links. A downside of unprefixed, general terms like "item" :confused: I add some more examples to verify here if our concept will work well on this "simple" page.

Currently:
* GettingStartedSection > 1 grid > n cards > n items

Options I see
1. GettingStartedSection > 1 card > n columns > n items (IMHO fine)
2. GettingStartedSection > 1 card > n items > n items (IMHO this will make problems)
3. GettingStartedSection > 1 card > n items > n actions (IMHO fine)
4. GettingStartedSection > 1 card > n items > n options (sounds like a dropdown, but we use this term in the UI)

We can align this also with the AddCards

Currently:
* Add card section > n cards > n items

Options I see:
1. Keep cards and items
2. cards and actions
3. cards and options

(Solved the `data-test` change request with Gajanan in slack.)

Reply:
It is `column` now.

Reply:
Changed the `data-test` attribute for now to `card ${id}` so that this matches also our code structure.

I also added a data-test wrapper for the getting started section and the add cards. So its possible to distinguish them via:

```
[data-test~=getting-started] [data-test~=card $id]
or for all getting started cards (up to 3 at the moment)
[data-test~=getting-started] [data-test~=card]
```

or

```
[data-test~=add-cards] [data-test~=card $id]
or for all add cards:
[data-test~=add-cards] [data-test~=card]
```

For the getting started section, this works for example via `cy.byTestID('card samples')`,  `cy.byTestID('card quick-start')` and  `cy.byTestID('card developer-features')` for the developer getting started sections.

/cc @gajanan-more